augly/image/functional.py

```diff
@@ -1056,11 +1057,17 @@ def overlay_emoji(
 
     @param opacity: the lower the opacity, the more transparent the overlaid emoji
 
-    @param emoji_size: size of the emoji is emoji_size * height of the original image
+    @param emoji_size: size of the emoji is emoji_size * height of the original image.
+        If set to a tuple, the value will be randomly chosen from that range [low, high)
+
+    @param x_pos: position of emoji relative to the image width. If set to a tuple, the
+        value will be randomly chosen from that range [low, high)
 
-    @param x_pos: position of emoji relative to the image width
+    @param y_pos: position of emoji relative to the image height. If set to tuple, value
+        will be randomly chosen from that range [low, high)
 
-    @param y_pos: position of emoji relative to the image height
+    @param seed: if provided, this will set the random seed to ensure
+            consistency between runs
```

Comment:
if we decide to continue to set the random parameters here instead of transforms.py, please ensure all docstrings are identical across both files. otherwise, revert these changes!